addons/block_code/ui/constants.gd

```diff
@@ -1,5 +1,7 @@
 extends Object
 
+const CURRENT_DATA_VERSION = 1
```

Comment:
Backtracking on my earlier thought, I wonder if this should still be `0` since we haven't changed the serialization yet. #107, for example, would bump this up. Or we just keep it `0` until we're ready to handle stability.

Reply:
Yes, let's keep it zero until we are ready to handle migrations.